src/Bindings.re

```diff
@@ -54,3 +54,10 @@ module LocalStorage = {
   [@mel.send] external removeItem: (t, string) => unit = "removeItem";
   external localStorage: t = "localStorage";
 };
+
+module Element = {
+  type t = Dom.element;
+
+  [@mel.send]
+  external scrollIntoView: (t, Js.t('options)) => unit = "scrollIntoView";
```

Comment:
Do you think it's worth adding types for `options`? Also, we can use records now instead of `Js.t`.